core/src/main/java/org/ehcache/Ehcache.java

```diff
@@ -781,7 +822,7 @@ public V apply(final K k, final V inCache) {
       }
     });
     try {
-      store.computeIfPresent(key, remappingFunction, REPLACE_FALSE);
+      store.compute(key, remappingFunction, REPLACE_FALSE);
```

Comment:
Thinking ahead a little bit, how do we think these compute methods will work on clustered stores? If compute() might necessarily end up being slower than computeIfPresent() then it might be worth complicating things here to decide which one to call on the store depending on whether "useLoaderInAtomics" is true and whether there is a cacheLoader.